src/main/java/my/project/lotto/domain/Machine.java

```diff
@@ -0,0 +1,27 @@
+package my.project.lotto.domain;
+
+import my.project.constants.Rule;
+
+import java.util.Collections;
+import java.util.Stack;
+
+
+/**
+ * Created : 2020-11-23 오후 2:30
+ * Developer : Seo
+ */
+public class Machine {
+    private final Stack<Integer> numbers;
+
+    public Machine() {
+        numbers = new Stack<>();
+        for (int i = Rule.MIN_NUMBER; i < Rule.MAX_NUMBER + 1; i++) {
+            numbers.push(i);
+        }
+        Collections.shuffle(numbers);
+    }
```

Comment:
```suggestion
    private static final List<Integer> numbers = new ArrayList<>();
    
    static {
        for (int i = Rule.MIN_NUMBER; i < Rule.MAX_NUMBER + 1; i++) {
            numbers.push(i);
        }
    }
```
하나의 로또를 만들 때 마다 계속 1 ~ 45 사의 수를 stack에다 초기화 하는 형태로 만들어진 상태입니다.
차라리 1 ~ 45를 담고 있는 리스트를 만든 다음에 새로운 로또를 만들 때 마다 Shuffle 후에 6개씩 뽑아주는건 어떨까요?